src/Staircase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+class Staircase
+{
+    public function print(int $size)
+    {
+        $staircase = [];
+        for($i = 1; $i <= $size; $i++) {
```

Comment:
Este `for` pode ser simplificado para `do { } while($size--)`

Reply:
Acabei de reparar que falei besteira; Pode ignorar esse daqui.

Note to self: todo code review deveria ser seguido de um banho.